packages/proxy/lib/http/util/prerequests.ts

```diff
@@ -275,6 +275,7 @@ export class PreRequests {
       proxyRequestReceivedTimestamp: performance.now() + performance.timeOrigin,
       timeout: setTimeout(() => {
         ctxDebug('Never received pre-request or url without pre-request for request %s after waiting %sms. Continuing without one.', key, this.requestTimeout)
+        debug('Never received pre-request or url without pre-request for request %s after waiting %sms. Continuing without one.', key, this.requestTimeout)
```

Comment:
What is the difference between ctxDebug and debug?

Reply:
`ctxDebug` is enabled when `cypress-verbose:proxy:http` is set which includes a lot of other debug information. I wanted an easier way to see just the timeout issues so I added this debug which is set when `cypress:proxy:http:util:prerequests` is set.